psi4/src/psi4/libfock/points.h

```diff
@@ -245,7 +245,7 @@ class RKSFunctions : public PointFunctions {
     size_t block_index() { return block_index_; }
 };
 
-class UKSFunctions : public PointFunctions {
+class PSI_API UKSFunctions : public PointFunctions {
```

Comment:
Were these psi_api leftover from the plugin implementation?

Reply:
Yes, they were. I will fix it